src/layouting/transformation.jl

```diff
@@ -242,11 +242,11 @@ end
 # PointTrans{N}(func::F) where {N, F} = PointTrans{N, F}(func)
 Base.broadcastable(x::PointTrans) = (x,)
 
-function apply_transform(f::PointTrans{N}, point::Point{N}) where N
+function apply_transform(f::PointTrans{N}, point::VecTypes{N}) where N
     return f.f(point)
 end
 
-function apply_transform(f::PointTrans{N1}, point::Point{N2}) where {N1, N2}
+function apply_transform(f::PointTrans{N1}, point::VecTypes{N2}) where {N1, N2}
```

Comment:
Do we actually need this anywhere? Since we've been fighting a lot with compile times and clarity of what types a function accepts, I started restricting the types more whenever possible.
This way, this method is compiled less and anyone calling `apply_transform` is forced to use less types.
Which I think is a good trade off for an internal function as this

Reply:
I believe it gets used for `Arrows` and some other things, there was definitely a reason I had this here although I don't remember exactly what it was now.

Reply:
I think that was the problem - anything with Vec caused a methoderror when it was being transformed.